spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/TraceFilterIntegrationTests.java

```diff
@@ -119,6 +120,8 @@ public void should_add_a_custom_tag_to_the_span_created_in_controller() throws E
 				.filter(span -> span.tags().containsKey("tag")).findFirst();
 		then(taggedSpan.isPresent()).isTrue();
 		then(taggedSpan.get()).hasATag("tag", "value");
+		then(taggedSpan.get()).hasATag("http.class.method", "deferred");
```

Comment:
I'd probably not use http namespace, since this isn't http (like http.method).

maybe mvc.controller.method or similar?